tensorboard/backend/http_util.py

```diff
@@ -184,6 +189,8 @@ def Respond(
     headers.append(("X-Content-Type-Options", "nosniff"))
     if content_encoding:
         headers.append(("Content-Encoding", content_encoding))
+    if filename:
+        headers.append(("Content-Disposition", 'filename="%s"' % filename))
```

Comment:
Can we use [`werkzeug.http.quote_header_value(filename)`][1] here?
Callers need to sanitize the logical filename, but shouldn’t need to
know about the RFC 2616 quoted-string grammar.

[1]: https://werkzeug.palletsprojects.com/en/1.0.x/http/#werkzeug.http.quote_header_value

Reply:
Ooh yes, thanks for pointing that out.  Done and test added.